### PR TITLE
docs: change the meetup url for Angular

### PR DESCRIPTION
**Pull request by santoshyadavdev**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/navigation.json [hunk 1]
```diff
@@ -833,7 +833,7 @@
           "tooltip": "Angular events around the world."
         },
         {
-          "url": "http://www.meetup.com/topics/angularjs/",
+          "url": "https://www.meetup.com/find/?keywords=angular",
```
Comment (gkalpak) [inline aio/content/navigation.json]: The `http --> https` change looo=ks fine, but the rest doesn't :grin:
The first URL shows ~800 groups, while the secind only 1 😕

Comment (santoshyadav198613) [reply aio/content/navigation.json]: The problem is by default is shows meetup groups filtered in your current location, let me check if it is possible to show all groups, in my case it shows all meetup groups in Pune with Angular tag.

Comment (gkalpak) [reply aio/content/navigation.json]: I see. I guess one could use https://www.meetup.com/find?&keywords=angular&radius=Infinity, but the first URL still seems to have much more info (and better organized). For example, you can get an overview of how many meetups/members there are, you can see them in a map, see relevant meetups near you, see most active meetups, etc.

What is your motivation for trying to switch to a different URL?

Comment (santoshyadav198613) [reply aio/content/navigation.json]: moving from angularjs to angular

Comment (gkalpak) [reply aio/content/navigation.json]: That's a noble goal :smiley:
However, I don't think that is the right way. I don't know much about meetup topics (maybe we could somehow rename `AngularJS` to `Angular` or have a topic alias).

Comment (santoshyadav198613) [reply aio/content/navigation.json]: yeah sure :) let me check if we can do this. Will check on meetup APIs

**Comment by santoshyadavdev** [comment]
Hi @gkalpak ,
I have just kept this from HTTP to https change. please check

**Review by gkalpak — APPROVED** [review]
LGTM

Reviewed-for: global-docs-approvers

**Pull request closed without merging**